packages/rpc-methods/src/permitted/common/snapInstallation.ts

```diff
@@ -3,7 +3,7 @@ import { InstallSnapsResult } from '@metamask/snaps-utils';
 import { isObject } from '@metamask/utils';
 import { ethErrors } from 'eth-rpc-errors';
 
-export { InstallSnapsResult } from '@metamask/snaps-utils';
+export type { InstallSnapsResult } from '@metamask/snaps-utils';
```

Comment:
This was necessary, because Vite (used by `wdio`) was complaining that `@metamask/snaps-utils` doesn't export `InstallSnapsResult` (which it doesn't, since it's just a type).